src/2_RTB/AnalisiDeiRequisiti/changelog.csv

```diff
@@ -2,4 +2,6 @@
 0.2,16-11-2023,"Stesura sezioni 1.1, 1.3, 2.1, 2.2 (Funzionalità del prodotto), 3.1, 3.2, modifiche a UC7.1, UC7.2 e UC9, forma tabellare per requisiti, aggiunta requisiti Opzionali, 4.2, 4.3, 4.4",Simone Caregnato,Analista
 0.3,17-11-2023,"Ulteriore dettaglio su sezioni 1.2, 2.2, 2.3, 3.1, 4.1",Nancy Kalaj,Analista
 0.4,19-11-2023,"Apportate modifiche a sezioni 2.1, 2.4, 3.2, e agli UC",Giacomo D'Ovidio, Analista
-0.5,19-11-2023,"Modificate sezioni 1.2, 2.2 e apportati cambiamenti sugli UC", Costantin Riccardo, Analista
\ No newline at end of file
+0.5,19-11-2023,"Modificate sezioni 1.2, 2.2 e apportati cambiamenti sugli UC", Costantin Riccardo, Analista
+0.6,22-11-2023, "Aggiunto diagramma UML UC", Giacomo D'Ovidio, Analista
+0.6,22-11-2023,"Modifcato diagramma casi d'uso",Costantin Riccardo, Analista
```

Comment:
due volte la stessa versione

Reply:
non siamo esattamente in questo caso? 

Da norme di progetto v 0.5
Due modifiche, fatte in momenti diversi, differiscono l’una dall’altra solo se
hanno scopi diversi. Ad esempio non è necessario incrementare la versione se
viene fatta una modifica alla stessa sezione in due giorni differenti; anche se
faccio una modifica, ed essa non viene approvata, non è necessario
incrementare la versione con le nuove modifiche proposte dal/dai
verificatore/i, dal momento che modifica e verifica “viaggiano”
parallelamente.

Reply:
Si ma in questo caso:
1. sono state fatte da persone diverse;
2. non si tratta di un lavoro iniziato e non finito, si tratta di un lavoro finito (`0.6`) e successivamente modificato per x ragione (`0.7`),

Se addirittura la "Modifica diagramma casi d'uso" di Riccardo e' stata fatta in seguito ad una revisione, allora quella riga non ci deve essere proprio.